### ISSUE TITLE
Add cli to create users

### ISSUE THREAD

**Issue by Thovi98** [SamR1/FitTrackee#319]
For now, Yunohost admins can install FitTrackee entirely from the Yunohost webadmin interface.

However, to create a user, they have to do it via browser on FitTrackee, and then set it as admin via cli.

Until https://github.com/SamR1/FitTrackee/issues/312 is implemented (if you are interested in it of course), it would be nice to let people create users via `ftcli`.

This way, the Yunohost package could create a user directly into its scripts and then set it as admin, without having to go to cli after installation.

It could look like `ftcli users create --username <user> --email <email> --password <password>`. And I suppose that a cli created user would automatically accept the privacy policy. (Although this § is just a suggestion of course 😁)

I don’t know if this make sense?

Related (but rather an alternative) to https://github.com/SamR1/FitTrackee/issues/228

**Comment by SamR1** [comment]
released in v0.7.15

**Note**: the command is: 
```shell
ftcli users create <username> --email <email> --password <password>
```
see [documentation](https://samr1.github.io/FitTrackee/cli.html#ftcli-users-create)

**Comment by Thovi98** [comment]
Works great, thank you so much!
